test/swaggers/specification/scenarios/resource-manager/Microsoft.Test/2016-01-01/examples/storageAccountPathParameterForwardSlash.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameters":{
+    "api-version": "2016-01-01",
+    "subscriptionId": "foo/foo1/foo2",
```

Comment:
we could make it a little bit more real, if we had another parameter other than subscriptionId, since subscriptionID does not follow this pattern.